extensions/exposition/components/identity.federation/manifest.toa.yaml

```diff
@@ -44,6 +44,10 @@ operations:
     output:
       identity:
         id: string
+        claim:
+          iss: string
+          aud: string
```

Comment:
We have `aud` as array of strings in the trust configuration, what would be this single sting - a matched audience?

Reply:
https://github.com/toa-io/toa/pull/656/commits/b538908c4e1a0fbe6dae5140adfcd561c8f0382b